#!/usr/bin/python3

def raise_exception():
    a = 9
    b = "abana"

    print(a + b)
